build.gradle

```diff
@@ -57,10 +59,11 @@ tasks.withType(JavaCompile) {
     options.compilerArgs += ['-Werror']
 }
 
-gradlePlugin {
-    // do not add new task to publish to plugins.gradle.org
-    automatedPublishing = false
-
+pluginBundle {
+    website = 'https://github.com/palantir/gradle-consistent-versions'
+    vcsUrl = 'https://github.com/palantir/gradle-consistent-versions'
+    description = 'Compact, constraint-friendly lockfiles for your dependencies'
+    tags = ['versions']
     plugins {
```

Comment:
Think you want to keep this block under `gradlePlugin` as it's a bit different (it configures the `java-gradle-plugin` plugin that generates the plugin descriptor files).
`pluginBundle.plugins` will actually be auto-populated from the ids defined in `gradlePlugin.plugins`, see the [example](https://plugins.gradle.org/docs/publish-plugin). Although you can still configure `pluginBundle.plugins` separately if you want to give them display names, though I don't think it's necessary.